calculator.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+ 
+all_data = {
+    "Transaction Type": "",
+    "Origin or Destination": "",
+    "Amount": "",
+    "Date": ""
+}
+ 
+def data():
+    # input either income or expense
+    choose = input("Income or Expense? ")
+    all_data.update({"Transaction Type": choose})
```

Comment:
you don't want to rely on your user to input exactly what you need. standardize to something based on user input and if the user doesn't input what you want, don't accept it.